src/style.css

```diff
@@ -1,3 +1,61 @@
-body {
+body, h1, h2, h3, p, ul {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  background-color: white;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  left: 0;
+  right: 0;
+  align-items: center
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  list-style: none;
+  text-decoration: none;
+  color: #000;
+  font-family: Roboto, sans-serif;
```

Comment:
Move to html selector and remove it from all other places.

Reply:
![1](https://github.com/mate-academy/layout_moyo-header/assets/144326045/92f956ef-8ae5-4e7a-8388-c4a2265dd84a)
![2](https://github.com/mate-academy/layout_moyo-header/assets/144326045/9bb4e8a0-d3be-4468-9fb7-d7cdf3b18406)
![3](https://github.com/mate-academy/layout_moyo-header/assets/144326045/91a85db8-fa86-4694-8cdc-8a5a5d90f02a)
what's wrong here?